common/lib/xmodule/xmodule/js/test_config.js

```diff
@@ -0,0 +1,8 @@
+// By default, fixtures are loaded from spec/javascripts/fixtures but in karma everything gets served from /base
+jasmine.getFixtures().fixturesPath = '/base/';
+
+// https://github.com/edx/js-test-tool/blob/master/js_test_tool/templates/jasmine_test_runner.html#L10
+// Stub out modal dialog alerts, which will prevent
+// us from accessing the test results in the DOM
+window.confirm = function(){return true;};
+window.alert = function(){return;};
```

Comment:
Can we not put this file in common and use it everywhere instead of having a copy for each suite?